main.py

```diff
@@ -45,6 +45,11 @@ def init():
     # Register exit handler
     atexit.register(on_exit)
 
+def os_pause():
+	if sys.platform == "win32" and not DUMB_MODE:
```

Comment:
__[flake8]__
* [E101] indentation contains mixed spaces and tabs
* [W191] indentation contains tabs

<!-- Comment by [SideCI](https://sideci.com) -->